src/map/packet_system.cpp

```diff
@@ -3857,8 +3857,27 @@ void SmallPacket0x05E(map_session_data_t* const PSession, CCharEntity* const PCh
             else
             {
                 // Ensure the destination exists
-                CZone* PDestination = zoneutils::GetZone(PZoneLine->m_toZone);
-                if (settings::get<bool>("main.ERA_CHOCOBO_ZONE_DISMOUNT") && PChar->isMounted() && PDestination && !PDestination->CanUseMisc(MISC_MOUNT))
+                // clang-format off
+                CZone*   PDestination   = zoneutils::GetZone(PZoneLine->m_toZone);
+                ZONEID   charZone       = (ZONEID)PChar->getZone();
+                bool     isInHomeNation = PChar->profile.nation == NATION_BASTOK ? charZone >= ZONE_BASTOK_MINES && charZone <= ZONE_PORT_BASTOK : false ||
+                                          PChar->profile.nation == NATION_WINDURST ? charZone >= ZONE_WINDURST_WATERS && charZone <= ZONE_WINDURST_WOODS : false ||
+                                          PChar->profile.nation == NATION_SANDORIA ? charZone >= ZONE_SOUTHERN_SANDORIA && charZone <= ZONE_PORT_SANDORIA : false;
+                bool     isInRentedCity = PChar->getCharVar("[Moghouse]Rent-a-room") == static_cast<int32>(zoneutils::GetZone(charZone)->GetRegionID());
```

Comment:
(again mixed casing with before, sometimes caps or not)

Reply:
Fixed